src/components/players/PlayerProfilePage.js

```diff
@@ -0,0 +1,83 @@
+import { Fragment, useCallback, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import { playerActions } from "../../store/player";
+import classes from "./PlayerProfilePage.module.css";
+import PlayerSeasonAverages from "./PlayerSeasonAverages";
+import PlayerStatsByGame from "./PlayerStatsByGame";
+import LoadingSpinner from "../UI/LoadingSpinner";
+import { getPlayerProfileData } from "../../services/playersProfileService";
+import { useNavigate } from "react-router-dom";
+import LogoComp from "../UI/LogoComp";
+
+const PlayerProfilePage = () => {
+  const dispatch = useDispatch();
+  const [playerData, setPlayerData] = useState();
+  const [isLoading, setIsLoading] = useState(false);
+  const seasonYear = useSelector((state) => state.season.season);
+  const params = useParams();
+  const playerName = params.playerName.replace("-", " ");
+  const navigate = useNavigate();
+
+  const backToPlayersHandler = () => {
+    navigate("/players");
+    dispatch(playerActions.initPlayerData());
+  };
+
+  const getPlayerData = useCallback(async (playerName, seasonYear) => {
+    try {
+      setIsLoading(true);
+      const data = await getPlayerProfileData(playerName, seasonYear);
+      setPlayerData(data);
```

Comment:
why do you need this stateVariable? If you save it to redux store, than it should be available there? 
maybe I'm mistaking, long time no redux :)

Reply:
It is possible to do it with a local state or with the redux states. In my opinion it looks cleaner and easier to understand it with the local state. Anyway, I changed it to redux for now and it will be great if you will look at it again after I upload it to GITHUB.